facts/server.py

```diff
@@ -180,19 +191,19 @@ def process_request(request, tenantid, serverid):
     data.insert(len(fact_attributes) - 1, datetime.datetime.today().isoformat())
 
     # Check data coherency of time stamps
-    if len(mredis.range(tenantid, serverid)) > 2:
+    if len(mredis.range(tenantid, serverid)) >= 2:
```

Comment:
Because I need to check time stamp when it is greater or equeal to 2. If range is 4, It also needs to check time stamps